src/app/filters/filter-field.ts

```diff
@@ -12,6 +12,7 @@ import { FilterQuery } from './filter-query';
 export class FilterField {
   id?: string;
   placeholder?: string;
+  primaryQueries?: FilterQuery[];
```

Comment:
The data in the first section and second section is always different. Filter will only be applied to the second section. First section will always remain as it is.